### PR TITLE
Special report gallery 

### PR DESCRIPTION
**Pull request by zeftilldeath**
Special report tone galleries were looking pretty special...

# See
screen shot 2017-05-12 at 14 19 31

screen shot 2017-05-12 at 14 43 26

Now I have made them look special in a nice way

# See
screen shot 2017-05-12 at 14 40 22

screen shot 2017-05-12 at 14 19 00

screen shot 2017-05-12 at 14 19 11

In doing this I have made some small changes to galleries as a whole. The l-side-margins now begin after the standfirst, which results in less banded and strange looking colour blocks.

# See
screen shot 2017-05-12 at 14 26 33

This also affects immersive overlays with a special report tone too, but more work should be done separately to further improve immersives with a special report tone. 

screen shot 2017-05-12 at 15 39 00

### CODE REVIEW

File: applications/app/views/fragments/galleryBody.scala.html [hunk 1]
```diff
@@ -8,42 +8,49 @@
...
-<div class="@RenderClasses(
-            Map("l-side-margins--paidgallery" -> isPaidContent(page)),
-            "l-side-margins", "l-side-margins--media", "l-side-margins--gallery"
-        )">
+<div class="gallery__main-content">
 
     <article id="article" class="@RenderClasses(
             Map("content--paidgallery" -> isPaidContent(page)),
-            "content", "content--media", "content--gallery", "tonal", "content--immersive", s"tonal--${toneClass(page.item)}"
+            "content", "content--media", "content--gallery", "tonal", "content--immersive", "tonal--tone-media", s"tonal--${toneClass(page.item)}"
```
Comment (NataliaLKB) [inline applications/app/views/fragments/galleryBody.scala.html]: Doesn't tonal--tone-media already get added with `s"tonal--${toneClass(page.item)}`. If not, what does get added?

File: applications/app/views/fragments/galleryHeader.scala.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 @import views.support.TrailCssClasses.toneClass
 
-<header class="content__head tonal__head tonal__head--@toneClass(gallery.item)">
+<header class="content__head tonal__head tonal__head--tone-media tonal__head--@toneClass(gallery.item)">
```
Comment (NataliaLKB) [inline applications/app/views/fragments/galleryHeader.scala.html]: Same here. Lets wait for the paid for fapi client change to happen and then we probably won't need to manually add this class.

**Review by NataliaLKB — APPROVED** [review]
Since this needs to go on Monday lets merge it, but next week lets take some time to do some of this properly

**Comment by paperboyo** [comment]
The second (welcomed!) change doesn’t work in Firefox (left):

capture

**Pull request merged**